deploy/helm/README.md

```diff
@@ -131,6 +131,7 @@ The values of the `OPERATOR_NAMESPACE` and `OPERATOR_TARGET_NAMESPACES` determin
 | `trivy.resources.limits.cpu`          | `500m`                             | The maximum amount of CPU allowed to run Trivy scanner pod.                                                                                                                                                                                                                                                                                                                           |
 | `trivy.resources.limits.memory`       | `500M`                             | The maximum amount of memory allowed to run Trivy scanner pod.                                                                                                                                                                                                                                                                                                                        |
 | `trivy.storageClassName`              | ``                                 | The name of the storage class to be used for Trivy server PVC.                                                                                                                                                                                                                                                                                                                        |
+| `trivy.podLabels`                     | ``                                 |                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
I wonder why there is no description for this setting.

Reply:
Thanks for pointing it out, I have updated the PR now.

In the future it would probably be nice to use something like https://github.com/norwoodj/helm-docs to make the developer experience better around the README files.